twoliter/embedded/imghelper

```diff
@@ -107,3 +107,89 @@ mkfs_data_xfs() {
   dd if=/dev/zero of="${bottlerocket_data}" bs=1M count="${size%?}"
   dd if="${bottlerocket_data}" of="${target}" conv=notrunc bs=1M seek="${offset}"
 }
+
+generate_ova() {
+  local os_vmdk data_vmdk
+  os_vmdk="${1:?}"
+  data_vmdk="${2:?}"
+
+  local os_image_publish_size_gib data_image_publish_size_gib
+  os_image_publish_size_gib="${3:?}"
+  data_image_publish_size_gib="${4:?}"
+
+  local ovf_template uefi_secure_boot sbkeys output_dir
+  ovf_template="${5:?}"
+  uefi_secure_boot="${6:?}"
+  sbkeys="${7:?}"
```

Comment:
This will eventually be a constant in `imghelper`, but I'll switch this when the secure boot code moves over.